deps/Makefile

```diff
@@ -485,12 +493,24 @@ LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=clang
 else
 # block default building of Clang
 LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=
+ifeq ($(LLVM_VER),svn)	# changed since LLVM 3.9
```

Comment:
Oh yeah, I fully agree with you there! Let's pray for some progress on #11754 :-) (although CMake has its own set of problems too...)